src/utils/axiosSetup.ts

```diff
@@ -48,5 +49,3 @@ const interceptor = (apiInstanceData: interceptorParams) => {
     }
   );
 };
-
-export default interceptor;
```

Comment:
I replaced the default export here because it was the only default export in the whole codebase.